ulid/__init__.py

```diff
@@ -261,6 +261,12 @@ def __get_pydantic_core_schema__(cls, source: Any, handler: GetCoreSchemaHandler
                     core_schema.no_info_plain_validator_function(ULID),
                 ]
             ),
+            serialization=core_schema.plain_serializer_function_ser_schema(
```

Comment:
I think better use this function from pydantic
[`to_string_ser_schema`](https://docs.pydantic.dev/latest/api/pydantic_core_schema/)

Reply:
Nice catch! Done.